### PR TITLE
start(#227): fetch topics for repos too, document /steps in steps/README.md

### PR DESCRIPTION
**Pull request by h1alexbel**
In this Pr I introduced a new property to fetch in `discover-repos.rb`, `topics` for further analysis with LLM/ML. Besides that, I've documented a high-level process that happens inside `steps`. It might be helpful for newcomers to this repo.

### CODE REVIEW

**Comment by h1alexbel** [comment]
@yegor256 please, approve the workflow run

**Comment by yegor256** [comment]
@h1alexbel something is wrong with the Markdown

**Comment by h1alexbel** [comment]
@yegor256 cleaned it

**Comment by yegor256** [comment]
@h1alexbel maybe it's better to keep all headers at the `##` level? Like it's done in the entire document now.

**Comment by h1alexbel** [comment]
@yegor256 yeah, good idea

**Comment by h1alexbel** [comment]
@yegor256 all headers in `steps/README.md` at the `##` level now

File: steps/README.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+## Gathering Steps
```
Comment (yegor256) [inline steps/README.md:1]: @h1alexbel why do we need two sections at the same level??

**Comment by yegor256** [comment]
@h1alexbel thanks!

**Pull request merged**